_since_project_start

```diff
@@ -0,0 +1,44 @@
+=> [[32m#<Story:0x00558144518950[0m
```

Comment:
esse arquivo aqui?

Reply:
Ops, sorry about that. It's an old idea about create a new js class to keep the done stories

Reply:
I think this last file was generated by the tests or something like this. We'll exclude them